adding thinking about dataset (#11)
carpentries-incubator/study-groups: _episodes/04-objectives.md

>   (26 March 2021 - 9 April 2021).
>   See the Episode Trial Run callout below for more details.
> - Read [Chapter 4: Designing Challenges][cdh-4] of the [Curriculum Development Handbook][cdh].
> - If your lesson will use an example dataset, in the Data Set section of your Lesson Design Notes

> >   briefly describe whatfeatures and requirements for an ideal dataset.

Reviewer: ```suggestion
>   briefly describe the features and requirements an ideal dataset would have.
```
